crates/topos-tce-broadcast/src/lib.rs

```diff
@@ -121,7 +121,7 @@ impl ReliableBroadcastClient {
     /// Aggregate is spawned as new task.
     // #[instrument(name = "ReliableBroadcastClient", skip_all)]
     pub fn new(
-        config: ReliableBroadcastConfig,
+        _config: ReliableBroadcastConfig,
```

Comment:
Turns out, we just needed the `sample` parameters when creating the `ReliableBroadcastClient`. I am not sure about that yet, seems a bit strange.